GANDLF/models/resnet.py

```
@@ -52,7 +52,7 @@ def __init__(self, parameters: dict, blockType, block_config):
         elif self.n_dimensions == 3:
             self.output_size = (1, 1, 1)
         else:
-            sys.exit("Only 2D or 3D convolutions are supported.")
+            logging.error("Only 2D or 3D convolutions are supported.")
```

Same as in the comment of `efficientnet.py` file - we should throw an error.